app/views/checkout/index.html.erb

```diff
@@ -20,8 +20,9 @@
                     <img class="media-object" src="http://icons.iconarchive.com/icons/custom-icon-design/flatastic-2/72/product-icon.png" style="width: 72px; height: 72px;">
                   </a>
                   <div class="media-body">
-                    <h4 class="media-heading"><a href="#"><%= order_item.item.name.humanize %></a></h4>
-                    <h5 class="media-heading"> by <a href="#">username</a></h5>
+                    <h4 class="media-heading"><%= link_to "#{order_item.item.name.humanize}", "#", data: {toggle: "modal", target: "#dish#{order_item.item.id}Modal"} %></h4>
+                    <h5 class="media-heading">
```

Comment:
This way of naming the internal link seems odd. I will let this pass but `#dish2Modal` feels weird. We don't use camel case in Id's or anchor links